src/components/layout/data/hooks.js

```diff
@@ -1,8 +1,6 @@
 import { useMemo } from 'react';
 import Color from 'color';
 
-import colors from '../../../colors.scss';
```

Comment:
By removing this import, where does the `colors.scss` file get used/imported? If its not imported anywhere, how do the CSS variables get created?

Related to this, I'm wondering if it might make sense to move the creation of these CSS variables into the top-level CSS entry point of the application (`src/index.scss`) to avoid needing to have a standalone `colors.scss` file to import the following:

```
@import "@edx/paragon/scss/core/core";
@import "@edx/brand/paragon/variables";
```

These are already included in `src/index.scss`, so I'm wondering if removing the seemingly duplicate import of at least `@import "@edx/paragon/scss/core/core";` helps cut down on duplicate styles included?

Reply:
Good callout, double checked it usage and found some additional locations where the fields were being used and refactored them similarly to the hooks file in the layout directory.

Also ended up deleting the colors file and migrating everything over to the suggested `src/index.scss`. Testing it looks like its still working as intended 👍🏽